routes/index.js

```diff
@@ -38,7 +38,9 @@ router.get('/events/:appId/exec', eventLogger.appExecEvents);
 router.get('/apps', appsController.list);
 router.get('/apps/new', appsController.new);
 router.get('/apps/:appId', appsController.show);
+router.get('/apps/update/:appId', appsController.showUpdatePage);
```

Comment:
I think standard REST conventions would have this as `/apps/:appId/edit`

Reply:
Agreed. I've changed the endpoint to `/apps/:appId/edit`, however, I'm still calling it `Update` because `edit` implies you can go into the container and make changes. 

Willing to discuss this on our next group call.